content/en/docs/measuring/metric-providers/carbon-intensity-elephant-machine.md

```
@@ -0,0 +1,160 @@
+---
+title: "Carbon Intensity - Elephant - Machine"
+description: "Documentation for CarbonIntensityElephantMachineProvider of the Green Metrics Tool"
+date: 2026-05-11T08:49:15+00:00
+weight: 221
+---
+
+### What it does
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)